podpac/core/coordinates/coordinates.py

```diff
@@ -525,24 +539,10 @@ def dims(self):
         return tuple(c.name for c in self._coords.values())
 
     @property
-    def shape(self):
-        """:tuple: Tuple of the number of coordinates in each dimension."""
-        
-        return tuple(c.size for c in self._coords.values())
-
-    @property
-    def ndim(self):
-        """:int: Number of dimensions. """
-        
-        return len(self.dims)
-
-    @property
-    def size(self):
-        """:int: Total number of coordinates."""
+    def idims(self):
+        """:tuple: Tuple of dimension index names."""

```

Comment:
It would be helpful to have an example here. We should document the use cases for dims, udims, and idims.  I put some in the Coordinates notebook